app/Helpers/database/game.php

```diff
@@ -68,6 +68,10 @@ function getGameMetadata(
         14 => "ORDER BY ach.Points DESC, ach.ID DESC ",
         5 => "ORDER BY ach.Title, ach.ID ASC ",
         15 => "ORDER BY ach.Title DESC, ach.ID DESC ",
+        6 => "ORDER BY 
+            CASE WHEN ach.type IS NULL THEN 2 WHEN ach.type = 'progression' THEN 0 WHEN ach.type = 'win_condition' THEN 1 ELSE 3 END, 
+            ach.ID ASC ",
+        16 => "ORDER BY ach.type, ach.ID DESC ",
```

Comment:
`16` should result in the same list as `6`, just reversed. I would expect the Case logic to be present here too. As it stands, because `progression` is alphabetically before `win_condition`, the win condition is not in the right place.

Reply:
Resolved in https://github.com/RetroAchievements/RAWeb/pull/1857/commits/dc335a38c85df9a807b4276afa4966fac32151cc.